src/configureWorkspace/configurePython.ts

```diff
@@ -111,6 +125,20 @@ function genDockerCompose(serviceName: string, ports: number[]): string {
         .replace(/\$ports\$/g, getComposePorts(ports));
 }
 
+function genDockerComposeDebug(serviceName: string, ports: number[], target: PythonTarget, projectType: PythonProjectType): string {
+    const args: string = (inferPythonArgs(projectType, ports) || []).join(' ');
+    const app: string = 'module' in target ? `-m ${target.module}` : target.file;
```

Comment:
should this also check if `target.module` has some value?

Reply:
Not really, there will always be a value in either "module" or "file" (we do require non-empty input), the question is which one of them to look at.